src/CaptiveLogin.vala

```diff
@@ -68,80 +67,77 @@ public class CaptiveLogin : Hdy.ApplicationWindow {
         };
         header.get_style_context ().add_class ("default-decoration");
 
-        notebook = new Granite.Widgets.DynamicNotebook () {
-            add_button_visible = false,
-            allow_drag = false,
-            allow_new_window = false,
-            allow_restoring = false,
-            expand = true,
-            tab_bar_behavior = Granite.Widgets.DynamicNotebook.TabBarBehavior.SINGLE
+        tabview = new Hdy.TabView () {
+            expand = true
+        };
+
+        var tabbar = new Hdy.TabBar () {
```

Comment:
IMHO this could probably use some start and end margins 🤷
```suggestion
        var tabbar = new Hdy.TabBar () {
            margin_start = 3,
            margin_end = 3,
```

Reply:
That's probably something that should be addressed in the stylesheet instead of individually in apps I think

Reply:
Filed https://github.com/elementary/stylesheet/issues/1267